ecosystem/sep-0006.md

```diff
@@ -190,7 +190,7 @@ Request Parameters:
 Name | Type | Description
 -----|------|------------
 `asset_code` | string | The code of the on-chain asset the user wants to get from the Anchor after doing an off-chain deposit. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
-`account` | `G...` string | The stellar account ID of the user that wants to deposit. This is where the asset token will be sent.
+`account` | `G...` string | The stellar account ID of the user that wants to deposit. This is where the asset token will be sent. Note that the account specified in this request could differ from the account authenticated via SEP-10.
```

Comment:
Is there text already somewhere in the SEPs that explain why the accounts can be different?

Reply:
No there isn't, but I could add a subsection under `Authentication`?

Reply:
I think it could be helpful, although maybe it is also obvious. I defer to you and will approve now and I can reapprove if you want to add a section.